ios/FluentUI/Avatar/Persona.swift

```diff
@@ -42,6 +42,11 @@ public protocol Persona {
 
     /// The subtitle value for the persona.
     var subtitle: String { get }
+
+    /// Loads the image asynchronously. This is called when setting up Persona row for presentation. Image caching should be handled in the implementation for optimal efficiency.
+    /// If the ``image`` is provided, it will be overrided by the loaded image when completion block is called, acting like a custom placeholder.
+    /// - Parameter completion: The completion block that returns the image
+    @objc optional func loadImageAsync(completion: @escaping (UIImage?) -> Void)
```

Comment:
nit: How about naming the func as "fetchImage" instead? (seems more aligned with the API naming convention that Apple is using)

Reply:
Sounds good, updated!